### PR TITLE
Add POSSIBLE Solutions Partner

### PR DESCRIPTION
**Pull request by stewsmith**

### CODE REVIEW

**Comment by stewsmith** [comment]
@kellyfinn @Hiliaryrobertson This is the new PR comparing to master.

**Comment by kellyfinn** [comment]
@dtothefp @stewsmith 
I don't see Possible appearing in the staging preview... sorry, not sure why. Can you take another look? Thanks.

**Comment by stewsmith** [comment]
@kellyfinn Updated. Try now.

File: website/partners/solutions/possible-uk/index.hbs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: partners
+title: "POSSIBLE"
+solutions: true
```
Comment (dtothefp) [inline website/partners/solutions/possible-uk/index.hbs]: @stewsmith @kellyfinn Is there a reason that POSSIBLE is in all caps? None of our other partners are I don't think

**Comment by dtothefp** [comment]
@stewsmith this has been updated to new Assemble syntax

**Comment by ihsekat** [comment]
@kellyfinn Hi Kelly, when you are creating these partner pages, could you add the following code to the .hbs file?

+TR_seo_title: "COMPANY_NAME - Optimizely"

That way the title won't default to "Optimizely: Make every experience count"

**Comment by ihsekat** [comment]
Or David, could you make it so the if +title exists, the default title becomes "+title - Optimizely"? Should I file a JIRA ticket for that?

**Comment by dtothefp** [comment]
@ihsekat @kellyfinn this is definitely a Marketing Eng task. Could you please put it in a JIRA ticket and we will move it into our sprint because we need to track these kinds of things in our sprint velocity. Thansk

**Comment by stewsmith** [comment]
@kellyfinn Could you take a look at this PR?

**Comment by kellyfinn** [comment]
This is ready, thanks. I'll follow up on a Jira ticket for the page name.
@dtothefp @stewsmith

**Comment by dtothefp** [comment]
resolved merge conflict

**Pull request merged**